projects/storyteller/webapp/proto/data.proto

```diff
@@ -61,15 +61,15 @@ message Screenshot {
 
 message StoryItem {
   reserved 2;
-  int64 timeMs = 1;
+  int64 time_ms = 1;
   string oneliner = 3;
   string note = 4;
   string screenshot_filename = 5;
   string id = 6;
 }
 
 message Story {
-  int64 startTimeMs = 1;
+  int64 start_time_ms = 1;
   int64 endTimeMs = 2;
```

Comment:
Ops, some of the fields are still in camel case:
https://github.com/hasadna/hasadna/blob/addingAngularMaterial/projects/storyteller/webapp/proto/data.proto#L48
https://github.com/hasadna/hasadna/blob/addingAngularMaterial/projects/storyteller/webapp/proto/data.proto#L73

Reply:
Also let me tell you why we need it.
When proto filed is written in camel case then case in actual data is broken.
And instead of `startTimeMs` we get `starttimems`.
So, since you was using proto with camel case, you also was getting data with broken case.
Example: https://github.com/hasadna/hasadna/blob/addingAngularMaterial/projects/storyteller/webapp/src/app/components/feed/feed.component.html#L14
(Should be `getTimeMs`)
So, when you fixed it, you'll receive data without the error, but error is still present in your code.
You need to find all the cases and fix it too.